index.js

```diff
@@ -45,52 +50,85 @@ function getLatestData(caseData) {
 }
 
 async function getCovidData(channelId, query) {
-  query = isEmpty(query) ? 'all' : query;
-  if (query == 'all') {
+  console.log(query);
+  query = isEmpty(query) ? ['all'] : (Array.isArray(query) ? query : [query]);
```

Comment:
I would make this into a second declaration, it's a wee bit dangerous to modify the function parameters. That, and nested ternaries can sometimes be kinda illegible (as a person who does that far too much 😛).

I'm thinking something like:
```suggestion
  let queryArray;
  if ( isEmpty(query)) {
    queryArray = ['all'];
  } else if(Array.isArray(query)) {
    queryArray = query;
  }
```

On the last line, since we already know the query is an array, we can just dump the data directly onto the variable.

Obviously from here you will have to change the references from `query` to "`queryArray`.